exercises/crypto-square/crypto_square_test.rb

```diff
@@ -1,105 +1,97 @@
 require 'minitest/autorun'
 require_relative 'crypto_square'
 
-class CryptoTest < Minitest::Test
-  def test_normalize_strange_characters
-    crypto = Crypto.new('s#$%^&plunk')
-    assert_equal 'splunk', crypto.normalize_plaintext
+# Common test data version: 2.0.0 bcdd704
+class CryptoSquareTest < Minitest::Test
+  def test_lowercase
+    # skip
+    crypto = Crypto.new('Hello')
+    assert_equal "hello", crypto.normalize_plaintext
   end
 
-  def test_normalize_uppercase_characters
+  def test_remove_spaces
     skip
-    crypto = Crypto.new('WHOA HEY!')
-    assert_equal 'whoahey', crypto.normalize_plaintext
+    crypto = Crypto.new('Hi there')
+    assert_equal "hithere", crypto.normalize_plaintext
   end
 
-  def test_normalize_with_numbers
+  def test_remove_punctuation
     skip
-    crypto = Crypto.new('1, 2, 3 GO!')
-    assert_equal '123go', crypto.normalize_plaintext
+    crypto = Crypto.new('@1, 2%, 3 Go!')
+    assert_equal "123go", crypto.normalize_plaintext
   end
 
-  def test_size_of_small_square
+  def test_empty_plaintext_results_in_an_empty_rectangle
     skip
-    crypto = Crypto.new('1234')
-    assert_equal 2, crypto.size
+    crypto = Crypto.new('')
+    assert_equal [], crypto.plaintext_segments
   end
 
-  def test_size_of_slightly_larger_square
+  def test_4_character_plaintext_results_in_an_2x2_rectangle
     skip
-    crypto = Crypto.new('123456789')
-    assert_equal 3, crypto.size
+    crypto = Crypto.new('Ab Cd')
+    assert_equal ["ab", "cd"], crypto.plaintext_segments
```

Comment:
So these tests seemed to have assertion that differed from what the canonical data wanted.
I went with the canonical data's choice of assertions, because I figured it was more up-to-date.